user_docs/en/userGuide.t2t

```diff
@@ -890,7 +894,12 @@ Selecting a form field and pressing enter or the Move to button moves to that fi
 %kc:beginInclude
 To report any comments for the currently focused cell, press NVDA+alt+c.
 %kc:endInclude
-All comments for the worksheet can also be listed in the NVDA Elements List.
+All comments for the worksheet can also be listed in the NVDA Elements List after pressing NVDA+f7.
+
+NVDA can also display a specific dialog for editing a certain comment.
+%kc:beginInclude
+To edit a certain comment, in a focused cell, press shift+f2.
+%kc:endInclude

```

Comment:
Everything reads well.  With shift+F2, is it worth clarifying that while NVDA overrides the dialog, shift+f2 is an Excel command (EG others differences are that shift+f2 isn't explained as an NVDA command in Input help in Excel, and you can't reassign it in Input Gestures).  Perhaps "NVDA overrides Excel's dialog for editing comments.... To add or edit a comment, in a focused cell, press shift+f2"?